### PR TITLE
Solve #48, Lesson 2: findLoop - Take in a linkedList, return true if the list loops on itself 

### PR DESCRIPTION
**Pull request by allopez7**

### CODE REVIEW

File: test/findloop.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+const g = {val:3, next:h};
+const f = {val:2, next:g};
+const e = {val:1, next:f};
+h.next = e;
+
+const j = {val:4};
+j.next = j;
+
+const i = {val:0};
+i.next = null;
```
Comment (songz) [inline test/findloop.js]: unnecessary.

File: test/findloop.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+const solution = require('../solutions/findloop.js');
+
+const d = {val:4};
+const c = {val:3, next:d};
+const b = {val:2, next:c};
+const a = {val:1, next:b};
+d.next = null;
```
Comment (songz) [inline test/findloop.js]: d.next is unnecessary.

File: solutions/findloop.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const findLoop = (obj, current)=>{
```
Comment (songz) [inline solutions/findloop.js]: current=object as default. You functon should only require 1 input, not 2

File: solutions/findloop.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const findLoop = (obj, current=obj)=>{
+  if(obj.next === current){
```
Comment (seemcat) [inline solutions/findloop.js:2]: Lint errors are when the left side of your terminal has red marks. Next time fix those lint errors and also do proper indentation.
if (obj.next == current) {

**Review by seemcat — APPROVED** [review]

**Pull request merged**